jobserver/github.py

```diff
@@ -353,13 +353,14 @@ def get_tag_sha(self, org, repo, tag):
 
         return r.json()["object"]["sha"]
 
-    def get_file(self, org, repo, branch):
+    def get_file(self, org, repo, branch, filepath=None):
+        filepath = filepath or "project.yaml"
```

Comment:
Any reason not to use `"project.yaml"` instead of `None` in the kwargs?

Reply:
No, I guess not...I just default to this pattern due to fear of mutable defaults, which "project.yaml" is not